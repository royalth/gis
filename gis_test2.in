7 6
1 2 
2 3
4 1 
5 1 
5 6 
7 5 
