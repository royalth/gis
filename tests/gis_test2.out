3
1 2 1
1 4 2
1 5 3
2 3 2
5 6 1
5 7 2
